internal/deploy/kyma.go

```diff
@@ -25,6 +26,12 @@ spec:
     enabled: {{ .Sync }}
 `
 
+var KymaGVR = schema.GroupVersionResource{
```

Comment:
If you want to avoid having dependencies to the lifecycle-manager, it is fine - If not I would suggest importing the GVK from the lifecycle-manager directly instead of hardcoding.

Reply:
Yes, I did it this way on purpose. Did not see the value of pulling in and maintaining a dependency for just a couple of strings.

But of course if in the future the CLI needs something else from the lifecycle manager it would be good to use that GVK.